crates/bevy_ui/src/layout/mod.rs

```diff
@@ -303,8 +309,23 @@ pub fn ui_layout_system(
 
     // compute layouts
     ui_surface.compute_window_layouts();
+}
+
+pub fn ui_nodes_system(
```

Comment:
I don't really see a good reason to make this pub. A public label might be good though.

Reply:
Yeah, I usually leave systems pub just because most of the systems in Bevy seem to be pub, but there isn't any need for it to be public here I agree.